_posts/2019-04-10-lecture-23.md

```diff
@@ -0,0 +1,357 @@
+---
+layout: distill
+title: "Lecture 23: Bayesian non-parametrics (continued)"
+description: Overview of inference in Dirichlet processes, topic models and the hierarchical Dirichlet process, and infinite latent variable models.
+date: 2019-04-10
+
+lecturers:
+  - name: Maruan Al-Shedivat
+    url: "https://www.cs.cmu.edu/~mshediva/"
+
+authors:
+  - name: Cormac OMeadhra
+    url: "#"
+  - name: Sumit Kumar
+    url: "#"
+  - name: Wentao Yuan
+    url: "#"
+
+editors:
+  - name: Lisa Lee
+    url: "http://leelisa.com"
+
+---
+
+<!--Recap: Chinese restaurant process, stick breaking construction-->
+## Recap
```

Comment:
This section looks empty?

Reply:
Sorry for the delayed response - I didn't realize I had my github notifications turned off. 

Anyway, we missed that section. I added it and pushed the update.